packages/hop-node/src/gasboost/GasBoostTransaction.ts

```diff
@@ -334,6 +334,11 @@ class GasBoostTransaction extends EventEmitter implements providers.TransactionR
   }
 
   getMaxGasPrice () {
+    // The xDai RPC endpoint sometimes returns bad data and should be capped lower.
+    if (this.chainSlug === Chain.xDai) {
+      const maxGasPriceGweiXdai = 90
```

Comment:
I think this should be moved to be part of `src/config`

```js
export const maxGasPriceGweiXdai = 90
```

Reply:
Good call. Fixed in 342a0a5a26ab86a06e3365b599b6e21f9031cb80